bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestManifest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Diagnostics;
+using NUnit.Framework;
+
+namespace ElectionGuard.Encrypt.Tests
+{
+    [TestFixture]
+    public class TestManifest
+    {
+        [Test]
+        public void Test_Can_Deserialize_Election_Description()
+        {
+            // Arrange
+            const string data = @"{""ballot_styles"":[{""geopolitical_unit_ids"":[""some-geopolitical-unit-id""],""object_id"":""some-ballot-style-id""}],""candidates"":[{""object_id"":""some-candidate-id-1""},{""object_id"":""some-candidate-id-2""},{""object_id"":""some-candidate-id-3""}],""contests"":[{""ballot_selections"":[{""candidate_id"":""some-candidate-id-1"",""object_id"":""some-object-id-affirmative"",""sequence_order"":0},{""candidate_id"":""some-candidate-id-2"",""object_id"":""some-object-id-negative"",""sequence_order"":1}],""electoral_district_id"":""some-geopoltical-unit-id"",""name"":""some-referendum-contest-name"",""number_elected"":1,""object_id"":""some-referendum-contest-object-id"",""sequence_order"":0,""vote_variation"":""one_of_m""},{""ballot_selections"":[{""candidate_id"":""some-candidate-id-1"",""object_id"":""some-object-id-candidate-1"",""sequence_order"":0},{""candidate_id"":""some-candidate-id-2"",""object_id"":""some-object-id-candidate-2"",""sequence_order"":1},{""candidate_id"":""some-candidate-id-3"",""object_id"":""some-object-id-candidate-3"",""sequence_order"":2}],""electoral_district_id"":""some-geopoltical-unit-id"",""name"":""some-candidate-contest-name"",""number_elected"":2,""object_id"":""some-candidate-contest-object-id"",""sequence_order"":1,""vote_variation"":""one_of_m""}],""election_scope_id"":""some-scope-id"",""end_date"":""2021-02-04T17:19:34Z"",""geopolitical_units"":[{""name"":""some-gp-unit-name"",""object_id"":""some-geopoltical-unit-id"",""type"":""unknown""}],""parties"":[{""object_id"":""some-party-id-1""},{""object_id"":""some-party-id-2""}],""start_date"":""2021-02-04T17:19:34Z"",""type"":""unknown""}";
```

Comment:
Probably want this line and line 28 to be consistent.

Reply:
That's fair but this is migrated code with the focus on packaging.

This is the `main` branches version.
https://github.com/microsoft/electionguard-cpp/blob/0bad662f3c6c32f4f7c257b2826d448930736594/bindings/netstandard/ElectionGuard/Tests/ElectionGuard.Tests/TestManifest.cs#L14

Reply:
yeah we need to remove this anyway and can be handled as part of #194 or #179